src/Screens/Main/Chat/MessageList/index.tsx

```diff
@@ -81,27 +84,57 @@ export function toRenderable(
     : messageList;
 }
 
-const MessageList = ({
+export const MessageList = ({
   messageList,
   getPreviousMessages,
   isLoading,
 }: Props) => {
   const messages = toRenderable(messageList, isLoading);
   const previousItem = messageList.length > 0 ? messageList[0].messageId : '0';
+  const dispatch = useDispatch();
 
   const getPreviousMessagesIfNotLoading = () => {
-    if (isLoading) {
+    if (isLoading || messageList.length < messageApi.MAX_MESSAGES_AT_ONCE) {
```

Comment:
What happens here?

Why we do nothing if len is less than max messages at once?

Reply:
If there is under that many messages, we dont want to fetch older messages. 
There can only be older messages if there is more than the fetch amount.

Reply:
Taa ei oo mulle ilmiselvaa et taa on oikein mut salee on :D, jos oot varma et menee oikein ni sit fine!

Must tuntu et tas on joku edge case mut ei kyl valtsiin oo